processor/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
```

Comment:
'processor' renamed to 'executor', so the whole 'processor' directory should be removed.